### PR TITLE
implemented email-only newsletter formular

### PR DESCRIPTION
**Pull request by dsumer**

### CODE REVIEW

File: src/pages/templates/forms/newsletter/emailOnly.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  const [email, setEmail] = useState("");
+  const [state, setState] = useState<"initial" | "submitting" | "success">(
+    "initial"
+  );
+  const [error, setError] = useState(false);
+
+  return (
+    <Flex w="100%" my={6} mx={2}>
+      <Flex
+        w={["100%", "95%", "80%", "70%"]}
```
Comment (hauptrolle) [inline src/pages/templates/forms/newsletter/emailOnly.tsx:23]: We prefer to use the object syntax for responsive values for a better readability. https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax

**Review by hauptrolle — CHANGES_REQUESTED** [review]
Super cool template!

I have just one small remark regarding the usage of the  responsive styles.

File: src/pages/templates/forms/newsletter/emailOnly.tsx (hunk 1 above)
Comment (dsumer) [reply src/pages/templates/forms/newsletter/emailOnly.tsx]: makes sense, I've changed it to the object syntax

**Review by hauptrolle — APPROVED** [review]

**Comment by hauptrolle** [comment]
@all-contributors please add @dsumer  for code

**Pull request merged**